src/services/activity/__tests__/ActivityTrackingService.test.ts

```
@@ -1,52 +1,56 @@
-      getCurrentActivity: jest.fn().mockResolvedValue(null),
-      setInterval: jest.fn(),
-    } as any;
+      start: mock(() => {}),
+      stop: mock(() => {}),
+      isTracking: mock(() => false),
+      getCurrentActivity: mock(() => Promise.resolve(null)),
+      setInterval: mock(() => {}),
+    };
 
-    (ActivityMonitor as jest.Mock).mockImplementation(() => mockMonitor);
+    (ActivityMonitor as any) = mock(() => mockMonitor);
```

Biome (2.1.2)

[error] The imported variable ActivityMonitor is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)